packages/frontend/containers/Coins.tsx

```diff
@@ -32,6 +35,7 @@ const COINS = {
     symbol: "USDC",
     icon: "Usd",
     stable: true,
+    decimals: 8,
```

Comment:
@kendricktan I thought USDC was 6 decimals?

Reply:
Not the best practice but i force pushed to master https://github.com/studydefi/dedge/commit/ff46a7048c5c5bde80dc123091068f6c5fd5fc1e